pages/home.md

```diff
@@ -88,14 +84,14 @@ permalink: /
           </div>
           <div class="column is-offset-1 is-5">
             <h2 class="title">
-              Using interoperable data standards
+              Linked Data: Interoperable data standards
             </h2>
             <p class="content is-large">
             Linked Data gives Solid 
-            a common way to describe things and 
-            how they relate to each other, in a way that other people and 
-            machines can understand. This means that the data stored by Solid
-            is portable and completely interoperable.
+            a <b>common</b> way to describe things and 
+            their relationships to each other such that other people and 
+            applications can understand. This means that the data stored by Solid
```

Comment:
Using applications instead of machines for continuity since the next section mentions applications accessing data as well as the bottom blurb about ecosystem of applications and data.